validation-test/stdlib/ArrayNew.swift.gyb

```diff
@@ -1011,8 +1003,6 @@ ArrayTestSuite.test("BridgedToObjC/Custom/BridgeBack/Cast") {
   expectEqual(idValue0, unsafeBitCast(a.object(at: 0) as AnyObject, to: UInt.self))
   expectEqual(idValue1, unsafeBitCast(a.object(at: 1) as AnyObject, to: UInt.self))
   expectEqual(idValue2, unsafeBitCast(a.object(at: 2) as AnyObject, to: UInt.self))
-
-  expectAutoreleasedKeysAndValues(unopt: (0, 3))
```

Comment:
Fun fact: if you remove the autoreleases these are looking for it gets reported as leaking a negative number of objects